processor/transform/tables-to-jsx.ts

```diff
@@ -0,0 +1,98 @@
+import { Parents, Table, TableCell, Text } from 'mdast';
+import { visit, EXIT } from 'unist-util-visit';
+import { Transform } from 'mdast-util-from-markdown';
+
+import { mdxJsx } from 'micromark-extension-mdx-jsx';
+import { fromMarkdown } from 'mdast-util-from-markdown';
+import { mdxJsxFromMarkdown } from 'mdast-util-mdx-jsx';
+import { phrasing } from 'mdast-util-phrasing';
+
+const visitor = (table: Table, index: number, parent: Parents) => {
+  let hasFlowContent = false;
+
+  const tableCellVisitor = (cell: TableCell) => {
+    if (!phrasing(cell.children[0])) {
+      hasFlowContent = true;
+      return EXIT;
+    }
+
+    visit(cell, 'text', (text: Text) => {
+      if (text.value.match(/\n/)) {
+        hasFlowContent = true;
+        return EXIT;
```

Comment:
i've always wanted a reason to use these actions 🥹

Reply:
Just start writing all you loops using `visit`